src/System.Management.Automation/engine/GetCommandCommand.cs

```diff
@@ -569,6 +567,80 @@ private void OutputResultsHelper(IEnumerable<CommandInfo> results)
 #endif
         }
 
+        /// <summary>
+        /// Creates the syntax output based on if the command is an alias, script, application or command.
+        /// </summary>
+        /// <param name="command">
+        /// CommandInfo object containing the syntax to be output.
+        /// </param>
+        /// <returns>
+        /// Syntax string cast as a PSObject for outputting.
+        /// </returns>
+        private PSObject GetSyntaxObject(CommandInfo command)
+        {
+            PSObject syntax = PSObject.AsPSObject(command.Syntax);
+
+            if (this.Name != null && !Array.Exists(this.Name, name => name.Equals(command.Name, StringComparison.InvariantCultureIgnoreCase)))
+            {
+                string aliasName = _nameContainsWildcard ? command.Name : this.Name[0];
+                
+                IDictionary<string, AliasInfo> aliasTable = SessionState.Internal.GetAliasTable();
+                foreach (KeyValuePair<string, AliasInfo> tableEntry in aliasTable)
+                {
+                    if (Array.Exists(this.Name, name => name.Equals(tableEntry.Key, StringComparison.InvariantCultureIgnoreCase)) && 
+                        tableEntry.Value.Definition == command.Name)
+                    {
+                        aliasName = tableEntry.Key;
+                        break;
+                    }
+                    else if (_nameContainsWildcard && tableEntry.Value.Definition == command.Name)
+                    {
+                        aliasName = tableEntry.Key;
+                        break;
+                    }
+                }
+
+                string replacedSyntax = string.Empty;
+                switch (command)
+                {
+                    case ExternalScriptInfo externalScript:
+                        replacedSyntax = string.Format(
+                            "{0} (alias) -> {1}{2}{3}",
+                            aliasName,
+                            string.Format("{0}{1}", externalScript.Path, Environment.NewLine),
+                            Environment.NewLine,
+                            command.Syntax.Replace(command.Name, aliasName));
+                        syntax = PSObject.AsPSObject(replacedSyntax);
```

Comment:
You repeat this line in each case, you can have it outside the switch statement

Reply:
They are slightly different as the output will need to vary depending on what is found for the alias/wildcard. If the found command is a script then it's path needs to be added along with an extra new line as the syntax property of commands already includes a leading new line.

Reply:
I'm referring specifically to:

```csharp
syntax = PSObject.AsPSObject(replacedSyntax);
```

Unless my eyes are missing something isn't it exactly the same in each case statement before the break?

Reply:
Good spot, I hadn't noticed that. I've fixed that now to only happen once after the switch.